### ISSUE TITLE
[Bug]: Adetailer models set to wrong folder.

### ISSUE THREAD

**Issue by gurukast** [BlafKing/sd-civitai-browser-plus#61]
### Describe the bug.

path is supposed to be Models\adetailer , but currently goes to Models\Other.
Was correct a couple of versions ago.

### Steps to reproduce the problem.

-click any adetailer model in browser

### Expected behavior

saved to adetailer model folder

### System info

* Extension version: v1.14.2 
version: [v1.6.0]  •  python: 3.10.6  •  torch: 2.0.1+cu118  •  xformers: 0.0.20  •  gradio: 3.41.2

### Console logs

```Shell
-
```

**Comment by BlafKing** [comment]
As far as I know I've never properly put adetailer models inside an Adetailer folder, since their content type is defined as "Other" and the code for that has always been:
```py
    elif content_type == "Other":
        folder = os.path.join(models_path,"Other")
```
Not sure which version properly put them inside the Adetailer folder but, I don't know how that could've happened 😅 


I'll add an extra detection to the Other content type to see if the word "Adetailer" is found in the description.
I'll make it so that if it is found, the Adetailer folder will get used 👍🏻

**Comment by BlafKing** [comment]
This is now fixed/added in v1.14.5! :)

I'll close this thread since it's now resolved 👍🏻
